index.d.ts

```diff
@@ -86,12 +87,15 @@ declare const globby: {
 
 	@param patterns - See the supported [glob patterns](https://github.com/sindresorhus/globby#globbing-patterns).
 	@param options - See the [`fast-glob` options](https://github.com/mrmlnc/fast-glob#options-3) in addition to the ones in this package.
-	@returns The matching paths.
+	@returns The matching paths - or the matching path's stats if the `stats` option is set.
 	*/
-	sync: (
+	sync: ((
+		patterns: string | readonly string[],
+		options: globby.GlobbyOptions & { stats: true }
+	) => Stats[]) & ((
```

Comment:
Why is this not just an overload?

Reply:
I am not sure but as this is declare as a key of a const object, you cannot just re-declare the key. There is a basic overload declared as the `globby` global result, but for the sync method, this is the best I could come up with.

Reply:
Alright. You still need to fix the formatting.

Reply:
Hum alright, I'll do that on Monday...

Reply:
Perhaps it could be changed into a regular overload? I'm unsure if it would be treated differently that way.

Apologies for butting in, I'm also editing this file for #177 and ran into this problem as well.

Reply:
Sorry for the delay, better ?